### ISSUE TITLE
Cannot extend from views/google-map/marker

### ISSUE THREAD

**Issue by LexLythius** [huafu/ember-google-map#58]
Fairly Ember rookie trying to implement custom markers.

According to the [wiki page](https://github.com/huafu/ember-google-map/wiki/Provided-Tools-and-Classes-%28API%29#markers):

> markerViewClass: name or class of the view to use for each marker, must extend google-map/marker view

So in my app I did
````javascript
import DefaultMarkerView from 'ember-google-map/views/google-map/marker';
````
And get an error: 
````
Uncaught Error: Could not find module `ember-google-map/views/google-map/marker` ...
````
whereas importing from `'ember-google-map/core/helpers'` does work.

I tried all sort of relative paths to get to it to no avail, so I compared things with other addons (like ember-cli-materialize). They have all their component classes and templates inside `addon` and inside `app` there are but mere exporting facades, e.g.:
````javascript
import MaterializeCollapsibleCard from 'ember-cli-materialize/components/md-card-collapsible';

export default MaterializeCollapsibleCard;
````
As a horrible expedient, copying google-map's `views` folder from `app` to `addon` fixes the import issue, so I guess the "facade-export from addon to app" approach should work.

Please correct me If I'm missing something here.

**Comment by huafu** [comment]
Yeah I need to move the views and more classes  at the right location. Look in the wiki page with example, there is one example there.

In brief, import the view as if it was in your app.

--
Huafu Gandon
http://huafu.github.io

**Comment by LexLythius** [comment]
Just submitted a PR about this

**Comment by LexLythius** [comment]
BTW I had tried importing the views from my app as if they were available locally before, but that didn't work either.

**Comment by Matt-Jensen** [comment]
+1 Ran into this issue myself.  I think there's also a lot of value to being able to extend the `google-map` component as well.  Real life use case was making a `google-heatmap` component, sadly ended up solving with copy/paste.

**Comment by huafu** [comment]
@LexLythius I've rebased, fixed and improved your PR, it's now in 0.0.25
